command/commands/team.py

```diff
@@ -106,26 +106,32 @@ def get_help(self):
         """Return command options for team events."""
         return self.help
 
-    def handle(self, command):
+    def handle(self, command, user_id):
         """Handle command by splitting into substrings and giving to parser."""
         logging.debug("Handling TeamCommand")
         command_arg = shlex.split(command)
-        args = self.parser.parse_args(command_arg)
+        args = None
+
+        try:
+            args = self.parser.parse_args(command_arg)
+        except SystemExit:
+            return self.help, 200
```

Comment:
Maybe at some point we want to have "standardized" codes in an enum or something - obviously we would use 200 here, but in other places it's possible the return code is ambiguous. Not really important for now though.